starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -717,6 +693,44 @@ sub check_dither {
             }
         }
     }
+
+
+
+
+    # For eng obs, don't have OR to specify dither, so stop before doing vs-OR comparisons
+    if ( $self->{obsid} =~ /^\d*$/){
+	return if ($self->{obsid} >= $ER_MIN_OBSID);
+    }
+
+    # Get the OR value of dither and compare if available
+    my $bs_val = $dither->{state};
+    my $or_val;
+    if (defined $self->{DITHER_ON}){
+        $or_val = $dthr_cmd{$self->{DITHER_ON}};
```

Comment:
This redirection is just confusing and makes the code harder to read.  At least make the variable name descriptive like `$translate_ON_OFF_to_ENAB_DISA`.